src/main/java/fr/silverpricing/api/model/Chambre.java

```diff
@@ -1,8 +1,12 @@
 package fr.silverpricing.api.model;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
 import lombok.*;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
 
 import javax.persistence.*;
+import java.time.Instant;
 import java.time.ZonedDateTime;
 import java.util.ArrayList;
 import java.util.List;
```

Comment:
:warning: The createChambre method violates the Single Responsibility Principle (SRP) by handling multiple responsibilities such as creating Price objects based on residence type. (Severity: Major)